tests/test_obo_reading.py

```diff
@@ -8,7 +8,6 @@
 
 directory = os.path.dirname(os.path.abspath(__file__))
 
-
 def test_read_taxrank_file():
```

Comment:
This test function also uses `.node` and is therefore causing test failures on networkx 2.5. Would you be able to update .node to .nodes here and apply the `pytest.importorskip`?

Sorry to make you fix things unrelated to the feature enhancement. I can also make these changes on master if they are becoming too burdensome.

Reply:
done. Travis seems to be happy now.